packages/appstore/src/components/onboarding/onboarding.tsx

```diff
@@ -30,7 +39,9 @@ const Onboarding = ({ contents, setIsTourOpen }: TOnboardingProps) => {
     const nextStep = () => {
         if (step < number_of_steps.length) setStep(step + 1);
         if (step === number_of_steps.length) {
-            setIsTourOpen(true);
+            setIsPreAppStore(true);
```

Comment:
setIsPreAppStore should be called once signup flow is completed on start trading button, no need to call it here again